.eslintrc.json

```diff
@@ -61,6 +61,12 @@
     "no-restricted-syntax": "off",
     "no-case-declarations": "off"
   },
+  "overrides": [{
+    "files": [ "./packages/wallets/**/src/**/*" ],
```

Comment:
```suggestion
    "files": [ "./packages/wallets/**/*" ],
```
this probably should be enough (but check please). Also, you can make local .eslintrc overrides, this will be better